### ISSUE TITLE
[Windows] Program does not have an icon

### ISSUE THREAD

**Issue by RattleSN4K3** [UltraStar-Deluxe/USDX#105]
Due to building a console app, the exe file does not have an icon. This results into desktopn icon, start menu entreis etc. not having a valid icon.

**Comment by basisbit** [comment]
this is not caused by the app being a console app but instead by lazarus 1.6 forgetting about the correct icon. Just before building, open the project settings and reselect the icon file which is in the src folder, then rebuild. Please confirm that this also works for you.

**Comment by RattleSN4K3** [comment]
Works. But the project file has to be updated somehow. Such workaround shouldn't be necessary for a clean build.

**Comment by RattleSN4K3** [comment]
Seems like the linking part is the problem. Having `ultrastardx.rc` doesn't link the file properly. With a renamed file it works well.

**Comment by RattleSN4K3** [comment]
cf3316dd8bacd12ea709b4e2d87a5ac97dc27f01.
Fixes the problem. Removed some unneeded files. The ultrastardx.res in the src folder is still generated everytime, not sure why it isn't created in the output directory. Therefore I also added it to the ignore files.

This also partially reverts changes done by the merge c4fc4aaf35a1c6469f747f50afcbac10be9e3342.

Can anyone test if this is also working on your system or using delphi?
